client/src/components/v2/CameraSettings.js

```diff
@@ -32,25 +43,55 @@ export default function CameraSettings({ device }) {
     }
   }, [selectedOverlay, setActiveOverlay]);
 
-  return (
-    <Card>
-      <Card.Body>
-        <Card.Title>{`${name} display`}</Card.Title>
-        {controls ? (
-          <RadioSelector
-            options={controls.overlays}
-            value={selectedOverlay}
-            onChange={setSelectedOverlay}
-          />
-        ) : (
-          <Card.Subtitle>Waiting for response...</Card.Subtitle>
-        )}
-      </Card.Body>
-      <Card.Footer>
-        <Card.Link href="#" onClick={handleSave}>Save</Card.Link>
-      </Card.Footer>
-    </Card>
-  );
+  const handleMessageSubmit = useCallback(() => {
+    sendMessage(message.current.value);
+    message.current.value = '';
+  }, []);
+
+  return ([
+    <div>
```

Comment:
What do the `<div>`s do around these cards?

Reply:
Since adjacent JSX elements must be wrapped in an enclosing tag, <div> separates them. I am also using the return argument as an array.

Reply:
This is a use case for [fragments](https://reactjs.org/docs/fragments.html)